src/styles/blocks/catalog.scss

```diff
@@ -18,8 +18,15 @@
     border-radius: 5px;
     margin-bottom: 48px;
 
+    transition: all 300ms;
```

Comment:
same

Reply:
but here I change two proporties transform and background-color at the same time.

Reply:
transition: color 300ms, transform 300ms;

And you can move duration to variable